NCEP/Readme.md

```diff
@@ -11,15 +14,27 @@ The NCEP GDAS Data Processing Utility is a Python script designed to download Gl
 
 ## Overview
 
-The National Centers for Environmental Prediction (NCEP) provides GDAS data that can be used for weather prediction and analysis. This utility simplifies the process of downloading GDAS data, extracting relevant variables, and converting it into a format compatible with the GraphCast weather prediction system.
+The National Centers for Environmental Prediction (NCEP) provides GDAS data that can be used for weather prediction and analysis. This repo simplifies the process of downloading GDAS data, extracting relevant variables, and converting it into a format compatible with the GraphCast weather prediction system. In addition, it automates running GraphCast with GDAS inputs on the AWS machine.
```

Comment:
"repo" -> "repository". "AWS machine" ->"AWS cloud"

Reply:
Also, can this script writes out "zarr" format gdas data?

Reply:
I did not work on zarr format yet, I can work on it in next PR, that would not take so much.